.dockerignore

```diff
@@ -0,0 +1,8 @@
+# We do not need the git repository in docker images.
+.git/
+
+# We do not need the typical build directories in docker images.
+.idea/
+.build/
+_build/
+cmake-build-debug/
```

Comment:
Should this be `cmake-build-*`? See comment on `.gitignore` for more details.

Reply:
Thoughts?

Reply:
SGTM.